packages/tools/devtools/devtools-core/api-report/devtools-core.api.md

```diff
@@ -108,24 +109,18 @@ export namespace ContainerList {
         type: typeof MessageType;
     }
     export interface MessageData {
-        containers: ContainerMetadata[];
+        containers: ContainerKey[];
```

Comment:
Note: this is a behavioral change. Since we no longer have to deal with 2 pieces of information (container ID and nickname), we can just pass an array of strings rather than an array of objects.